extmod/modnetwork.c

```diff
@@ -89,9 +89,52 @@ STATIC mp_obj_t network_route(void) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_0(network_route_obj, network_route);
 
+
+char mod_network_country_code[2] = "XX";
```

Comment:
this variable definition should go at the top of the file, just before the start of function definitons